packages/components/src/components/Forms/FormText.tsx

```diff
@@ -0,0 +1,167 @@
+import { Text } from 'react-native'
```

Comment:
After mulling it over a bit, I don't think I'm a fan of the Forms folder vs keeping the components file structure nice and flat of one folder, one component, no nesting/depth. Especially since some of the "forms" components are not necessarily intrinsically tied to form use cases. For example: secure messaging using text inputs and text boxes or claims using an individual Checkbox for acknowledgement or Rx Refill sort/filter where none feels like a "form" per se.

Not sure I have any good suggestions for renaming this file to be more generalized (maybe something like `sharedElements`?), but am thinking it preferable to flatten Checkbox/CheckboxGroup back into the `components` folder and shuffle this off to a new utils file.

Reply:
Moved Checkbox and CheckboxGroup back into components folder and created a new `shared` folder to contain FormText.tsx